### ISSUE TITLE
Audit #1 – Login Page – Input for text labels

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4319]
Connected to Audit 1, see #4300 
1.1 Login Page, URL: https://ocis-a11y.owncloud.works/

# Issue

Inputs for username and password have only placeholders as their accessible name, which is supported in most assistive technology but nevertheless a violation of WCAG 1.3.1/BITV 1.3.1h/WCAG 3.3.2/BITV 3.3.2a in my understanding.

# Remediation

Technique H44: https://www.w3.org/TR/WCAG20-TECHS/H44.html

**Comment by pascalwengerter** [comment]
Fixed via https://github.com/owncloud/ocis/pull/1794 and https://github.com/owncloud/ocis/pull/1811 in oCIS (since it's the IDP extension)
